ospfd/ospf_vty.c

```diff
@@ -4335,190 +4335,158 @@ static void show_ip_ospf_neighbour_header(struct vty *vty)
+static void show_ip_ospf_neighbour_brief(struct vty *vty,
+					 struct ospf_neighbor *nbr,
+					 struct ospf_neighbor *prev_nbr,
+					 json_object *json, bool use_json)
 {
-	struct route_node *rn;
-	struct ospf_neighbor *nbr, *prev_nbr = NULL;
 	char msgbuf[16];
 	char timebuf[OSPF_TIME_DUMP_SIZE];
 	json_object *json_neighbor = NULL, *json_neigh_array = NULL;
-	struct timeval res = {.tv_sec = 0, .tv_usec = 0};
+	struct timeval res;
 	long time_val = 0;
 	char uptime[OSPF_TIME_DUMP_SIZE];
 
-	for (rn = route_top(oi->nbrs); rn; rn = route_next(rn)) {
-		if ((nbr = rn->info)) {
-			/* Do not show myself. */
-			if (nbr == oi->nbr_self)
-				continue;
-			/* Down state is not shown. */
-			if (nbr->state == NSM_Down)
-				continue;
-
-			if (nbr->ts_last_progress.tv_sec
-			    || nbr->ts_last_progress.tv_usec)
-				time_val = monotime_since(
-						   &nbr->ts_last_progress, &res)
-					   / 1000LL;
+	if (nbr->ts_last_progress.tv_sec || nbr->ts_last_progress.tv_usec)
+		time_val =
+			monotime_since(&nbr->ts_last_progress, &res) / 1000LL;
 
-			if (use_json) {
-				char neigh_str[INET_ADDRSTRLEN];
+	if (use_json) {
+		char neigh_str[INET_ADDRSTRLEN];
 
-				if (prev_nbr
-				    && !IPV4_ADDR_SAME(&prev_nbr->src,
-						       &nbr->src)) {
-					/* Start new neigh list */
-					json_neigh_array = NULL;
-				}
+		if (prev_nbr && !IPV4_ADDR_SAME(&prev_nbr->src, &nbr->src)) {
+			/* Start new neigh list */
+			json_neigh_array = NULL;
+		}
 
-				if (nbr->state == NSM_Attempt
-				    && nbr->router_id.s_addr == INADDR_ANY)
-					strlcpy(neigh_str, "neighbor",
-						sizeof(neigh_str));
-				else
-					inet_ntop(AF_INET, &nbr->router_id,
-						  neigh_str, sizeof(neigh_str));
+		if (nbr->state == NSM_Attempt &&
+		    nbr->router_id.s_addr == INADDR_ANY)
+			strlcpy(neigh_str, "neighbor", sizeof(neigh_str));
+		else
+			inet_ntop(AF_INET, &nbr->router_id, neigh_str,
+				  sizeof(neigh_str));
 
-				json_object_object_get_ex(json, neigh_str,
-							  &json_neigh_array);
+		json_object_object_get_ex(json, neigh_str, &json_neigh_array);
 
-				if (!json_neigh_array) {
-					json_neigh_array =
-						json_object_new_array();
-					json_object_object_add(
-						json, neigh_str,
-						json_neigh_array);
-				}
+		if (!json_neigh_array) {
+			json_neigh_array = json_object_new_array();
+			json_object_object_add(json, neigh_str,
+					       json_neigh_array);
+		}
 
-				json_neighbor = json_object_new_object();
+		json_neighbor = json_object_new_object();
 
-				ospf_nbr_ism_state_message(nbr, msgbuf,
-							   sizeof(msgbuf));
+		ospf_nbr_ism_state_message(nbr, msgbuf, sizeof(msgbuf));
 #if CONFDATE > 20230321
-CPP_NOTICE("Remove show_ip_ospf_neighbor_sub() JSON keys: priority, state, deadTimeMsecs, address, retransmitCounter, requestCounter, dbSummaryCounter")
+		CPP_NOTICE("Remove show_ip_ospf_neighbor_sub() JSON keys: priority, state, deadTimeMsecs, address, retransmitCounter, requestCounter, dbSummaryCounter")
```

Comment:
whitespace change, I'd say drop it.

Reply:
The rest of  the lint errors can be ignored since they are not part of the change.